### PR TITLE
Search results table update

### PR DESCRIPTION
**Pull request by raivop**

### CODE REVIEW

File: grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy [hunk 1]
```diff
@@ -538,13 +538,88 @@ class OccurrenceTagLib {
...
+    /**
+     * Provides custom formatting per occurrence property (key).
+     *
+     * @attr value REQUIRED
+     * @attr key REQUIRED
+     * @return formatted value
+     */
+    def formatValue = { value, key ->
+        if(key == 'eventDate') {
+            return g.formatDate(date: new Date(value), format:"yyyy-MM-dd")
```
Comment (timo-piirmann) [inline grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: Space after "format:"

File: grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy [hunk 2]
```diff
@@ -538,13 +538,88 @@ class OccurrenceTagLib {
...
+        } else if(key == 'collectors' && value.size() > 2) {
+            return value[0..1].join(', ') + ', ...'
+        } else if(key == 'collectors') {
+            return value[0..-1].join(', ') // XXX funny
```
Comment (timo-piirmann) [inline grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: I think it's nicer "if key == 'collectors' " is checked once and then value.size() check is nested inside it

File: grails-app/assets/stylesheets/elurikkus-search-table.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/* Occurrence search results table container */
+.search-results {
+    clear: both;
+    overflow: hidden;
+    overflow-x: auto;
+}
+
+/* Occurrence search results table */
+.search-results-table {
+}
```
Comment (timo-piirmann) [inline grails-app/assets/stylesheets/elurikkus-search-table.css]: Empty css class?

File: grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy (hunk 1 above)
Comment (raivop) [reply grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: Good catch! I copy-pasted that big.

File: grails-app/assets/stylesheets/elurikkus-search-table.css (hunk 3 above)
Comment (raivop) [reply grails-app/assets/stylesheets/elurikkus-search-table.css]: Indeed it is.

**Pull request merged**